### ISSUE TITLE
kmath-core for iOS

### ISSUE THREAD

**Issue by iliamikulik** [mipt-npm/kmath#464]
Hello!

We're working on kotlin MPP library for mobile platforms, and we'd like to use your `kmath` library in our project. To be able to use it properly we need it to support iOS native targets such as `iosX64` and `iosArm64`.

I'm wondering if you have plans to implement `iosX64` and `iosArm64` target for your library anytime soon? As I get it, you already have a `macosX64` target.

I cloned the project and added these targets manually, and it was successful. But I see it should be done in `ru.mipt.npm.gradle` project, which might cause problems since this gradle plugin is also used in other projects.

We'd like to offer our help in solving the issue and make a pull request with changes in this project or in `ru.mipt.npm.gradle`.

**Comment by CommanderTvis** [comment]
I don't see any restrictions, except the build time. Consider adding these targets into https://github.com/mipt-npm/gradle-tools

**Comment by CommanderTvis** [comment]
Waiting for merge of https://github.com/mipt-npm/gradle-tools/pull/37. After that, Github Actions script will be modified a bit, and the next dev release will support iOS.

**Comment by altavir** [comment]
Yep. @CommanderTvis is a hero. It works. I need some fiddling with signing to publish it to central. It will be available in the next patch release.

**Comment by iliamikulik** [comment]
Hi! Sorry for the late answer, but I want to thank you so much for the work done! I checked it works

**Comment by and-marsh** [comment]
Hey, hope you're doing great!

We have been using your `0.3.1-dev-10` dev version for quite a while because some iOS targets are still missing in the latest release version. 

But since today we can't download the lib anymore as it requires authorization. Is there any chance that you could publish the release version with all iOS targets included?

**Comment by and-marsh** [comment]
Hey, thanks for looking into this!

Seems I was looking in the wrong Maven repo all this time. I was trying downloading `kmath` from [here](https://repo1.maven.org/maven2/space/kscience/kmath-core/0.3.1/kmath-core-0.3.1.module), and it's obvious some of the targets are missing there, while I should have checked out the [repo.kotlin.link](https://github.com/Heapy/repo.kotlin.link) instead.
 
The  `0.3.1` from `repo.kotlin.link` works for me just fine.

Thank you!
